packages/botonic-react/src/components/message.jsx

```diff
@@ -31,12 +32,17 @@ export const Message = props => {
   )
   if (isBrowser()) {
     useEffect(() => {
-      let decomposedChildren = decomposeComponent(textChildren)
+      let decomposedChildren = json
       let message = {
         id: state.id,
         type,
         data: decomposedChildren ? decomposedChildren : textChildren,
         from,
+        buttons: buttons.map(b => ({
+          payload: b.props.payload || b.props.path,
```

Comment:
path is not the same as payload, did you test this?

Reply:
I will test it!